lib/IRGen/IRGenFunction.h

```diff
@@ -226,6 +226,8 @@ class IRGenFunction {
                                  const llvm::Twine &name = "");
   void emitDeallocateDynamicAlloca(StackAddress address);
 
+  llvm::Value *emitStackAlloc(llvm::Value *size, llvm::Value *align);
```

Comment:
> Can you add a comment explaining this function, specifically that it will return nullptr if the size is too large.

I don't object to comments as a rule, but the other members in this header don't have comments or indicate when they might return any given value, so I'm not certain what makes this proposed member special. Can you elaborate?